packages/spear-cli/src/browser/InMemoryMagic.ts

```diff
@@ -166,6 +166,30 @@ export default async function inMemoryMagic(
   // generate static routing files.
   state.pagesList = await generateAliasPagesFromPagesList(state, jsGenerator);
 
+  // Embed assets
+  const asettsUrlAndRaw: {[key: string]: string} = {};
+  for (const assetFile of state.out.assetsFiles) {
+    const orgPath = assetFile.filePath.replace(settings.srcDir[0], "").replace(/\/\//, "/").replace(/^\//, "");
+    const orgURL = new URL(orgPath, window.location.href).href;
+    asettsUrlAndRaw[orgURL] = assetFile.rawData.toString();
+  }
+  const embedPageList = [] as Component[];
+  for (const page of state.pagesList) {
+    const parsedNode = embedAssets(
+      state,
+      asettsUrlAndRaw,
+      page.node.childNodes as Element[]
+    ) as Element[];
+    embedPageList.push({
+      fname: page.fname,
+      rawData: parsedNode[parsedNode.length > 1 ? 1 : 0].outerHTML,
```

Comment:
[Question]
Does that mean that whenever there is more than one `parsedNode` the second value is used?  🤔 
(not `parsedNode[parsedNode.length]` but `parsedNode[parsedNode.length > 1 ? 1 : 0]` ? )

Reply:
Original HTML might have `<!DOCTYPE html>`.
If content has this element, this `parsedNode` has two element. So we need use the second one. (First `<!DOCTYPE html>` doesn't have child.

As you mentioned, we can use `parsedNode[parsedNode.length]`.

Reply:
I addressed this 🙏🏼